bot.py

```
@@ -274,14 +286,27 @@ async def on_message(message):
             urls = re.findall(url_pattern, message.content)
 
             if urls:
-                # Process the first URL found
-                url = urls[0]
-                logger.info(f"Found URL in message {message.id}: {url}")
-
-                # Create a background task to process the URL
-                asyncio.create_task(process_url(message.id, url))
+                # Import config for limits
+                import config
+                
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)